fastlane/lib/fastlane/actions/get_github_release.rb

```diff
@@ -129,6 +129,7 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :api_token,
                                        env_name: "FL_GITHUB_RELEASE_API_TOKEN",
                                        sensitive: true,
+                                       default_value: ENV["GITHUB_API_TOKEN"],
```

Comment:
```suggestion
                                       code_gen_sensitive: true,
                                       default_value: ENV["GITHUB_API_TOKEN"],
                                       default_value_dynamic: true,
```

Good call! Can we also add these too? ^